setup.cfg

```diff
@@ -51,7 +51,7 @@ exclude = tests/
 follow_imports = silent
 
 [tool:pytest]
-testpaths = tests
+testpaths = src/ tests/
```

Comment:
Good to know. I'm unsure how I feel about this...

Reply:
I'm not a big fan of this, but I couldn't figure out another way to get `black` working without rewriting a lot of the CI configuration.

I personally would just call each utility separately instead of using `pytest` to run them all, but that seems like too large of a change when I was just trying to get the check to work.